web/src/components/CleanupDataTable.vue

```diff
@@ -32,7 +33,8 @@ export default {
     },
     columns() {
       const rows = this.dataset.sourcerows;
-      const f = v => (typeof v === 'number' ? v.toFixed(3) : v);
+      const nf = format('.4e');
+      const f = v => (typeof v === 'number' || (v && !Number.isNaN(v)) ? nf(parseFloat(v)) : v);
```

Comment:
I'd rather something like this, which will only use scientific notation when normal rendering of X significant digits is too long (i.e. really high or really tiny):

```
nf = n => n.toPrecision(6).length <= 10 ? n.toPrecision(6) : n.toExponential(4);
```

Currently has unnecessary use of exponential:

![image](https://user-images.githubusercontent.com/81305/67400712-0929e080-f57c-11e9-8bbe-094b60187757.png)